IOMC/ParticleGuns/src/RandomtXiGunProducer.cc

```diff
@@ -35,6 +35,7 @@ RandomtXiGunProducer::RandomtXiGunProducer(const edm::ParameterSet& pset) :
    fMaxt = pgun_params.getParameter<double>("Maxt");
    fMinXi= pgun_params.getParameter<double>("MinXi");
    fMaxXi= pgun_params.getParameter<double>("MaxXi");
+   fLog_t= pgun_params.getUntrackedParameter<bool>("Log_t",false);
```

Comment:
@mundim @civanch as this flag determines which kind of distribution is really simulated, would not be better to have it as a tracked parameter?

Reply:
By default, it assumes a linear distribution. The untracked parameter was used to keep backward compatibility with other config which do not use this parameter.